src/main/java/uk/gov/justice/hmpps/prison/service/MovementsService.java

```diff
@@ -266,7 +266,7 @@ public TransferSummary getTransferMovementsForAgencies(final List<String> agency
                         .courtDescription((String) e.get("courtDescription"))
                         .eventSubType((String) e.get("eventSubType"))
                         .eventDescription((String) e.get("eventDescription"))
-                        .hold((boolean) e.get("hold"))
+                        .hold("Y".equals(e.get("holdFlag")))
                         .build()
```

Comment:
Should "holdFlag" be "hold"? That's the name its being returned from the repository as.